lib/ViewModels/ChartPanelViewModel.js

```diff
@@ -49,6 +57,16 @@ var ChartPanelViewModel = function(options) {
 
     updateMaxDimensions();
 
+    knockout.getObservable(this, 'isVisible').subscribe(function(isVisible) {
+        var cesiumContainer = this.terria.currentViewer.viewer.container;
```

Comment:
Thanks @chloeleichen! Unfortunately this doesn't work with Leaflet... (it doesn't have a `viewer` property) - any ideas?